src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -1490,162 +1369,6 @@ void shouldSuccessfullyMapToHmiFormatWhenUpdateRequestHasNoRelatedPartyDetailsPr
-            HearingEntity hearingEntity = generateHearingEntity(hearingId, PutHearingStatus.LISTED.name(),
-                                                                versionNumber
-            );
-            addHearingResponses(hearingEntity, 1, true, 1, -1);
-            when(hearingRepository.findById(hearingId)).thenReturn(Optional.of(hearingEntity));
-            when(hearingRepository.existsById(hearingId)).thenReturn(true);
-            ActualHearingEntity actualHearingEntity = mock(ActualHearingEntity.class);
-            when(actualHearingEntity.getHearingResultType()).thenReturn(CANCELLED);
-            when(actualHearingRepository.findByHearingResponse(any(HearingResponseEntity.class)))
-                .thenReturn(Optional.of(actualHearingEntity));
-            ActualHearingDayEntity actualHearingDay = mock(ActualHearingDayEntity.class);
-            when(actualHearingDayRepository.findByActualHearing(any(ActualHearingEntity.class)))
-                .thenReturn(Optional.of(actualHearingDay));
-            Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
-                .hearingCompletion(hearingId));
-            assertEquals(HEARING_ACTUALS_UN_EXPRECTED, exception.getMessage());
-        }
-
-        @Test
-        void shouldInvokeUpdateCompletionStatusForActualHearing() {
-            final long hearingId = 2000000000L;
-            UpdateHearingRequest hearingRequest = TestingUtil.updateHearingRequest();
-            final int versionNumber = hearingRequest.getRequestDetails().getVersionNumber();
-            when(hearingRepository.getStatus(hearingId)).thenReturn(PutHearingStatus.LISTED.name());
-            HearingEntity hearingEntity = generateHearingEntity(hearingId, PutHearingStatus.LISTED.name(),
-                                                                versionNumber
-            );
-            addHearingResponses(hearingEntity, 1, true, 1, -1);
-            when(hearingRepository.findById(hearingId)).thenReturn(Optional.of(hearingEntity));
-            when(hearingRepository.existsById(hearingId)).thenReturn(true);
-            ActualHearingEntity actualHearingEntity = mock(ActualHearingEntity.class);
-            when(actualHearingEntity.getHearingResultType()).thenReturn(CANCELLED);
-            when(actualHearingRepository.findByHearingResponse(any(HearingResponseEntity.class)))
-                .thenReturn(Optional.of(actualHearingEntity));
-            when(actualHearingDayRepository.findByActualHearing(any(ActualHearingEntity.class)))
-                .thenReturn(Optional.empty());
-            ResponseEntity responseEntity = hearingManagementService.hearingCompletion(hearingId);
-            verify(hearingRepository, times(1)).save(any(HearingEntity.class));
-            assertNotNull(responseEntity);
-            assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
-        }
-    }
-
```

Comment:
part of hearings actual endpoint - needs to be re instated

Reply:
Retrieved